### PR TITLE
Integrations update

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

This PR tries to sync up integrations-extras with integrations-core integrations layout.

### Motivation

We want to pull info from integrations-extras integrations in DD doc

### TO DO

* Find a logo for each integrations and put in in documentation repo

### CODE REVIEW

File: filebeat/README.md [hunk 1]
```diff
@@ -29,4 +30,21 @@ When you run `datadog-agent info` you should see something like the following:
 
 ## Compatibility
 
-The filebeat check is compatible with all major platforms
+The Filebeat check is compatible with all major platforms
```
Comment (jeremy-lq) [inline filebeat/README.md]: Missing a period (looks to be the case throughout the PR).

File: filebeat/README.md [hunk 2]
```diff
@@ -29,4 +30,21 @@ When you run `datadog-agent info` you should see something like the following:
...
+
+## Data Collected
+### Metrics
+See [metadata.csv](https://github.com/DataDog/integrations-extras/blob/master/filebeat/metadata.csv) for a list of metrics provided by this integration.
+
+### Events
+The Filebeat check does not include any event at this time.
```
Comment (jeremy-lq) [inline filebeat/README.md]: s/event/events/g

File: filebeat/README.md [hunk 3]
```diff
@@ -29,4 +30,21 @@ When you run `datadog-agent info` you should see something like the following:
...
+
+### Service Checks
+The Filebeat check does not include any service check at this time.
```
Comment (jeremy-lq) [inline filebeat/README.md]: s/service check/service checks/g

File: filebeat/manifest.json [hunk 4]
```diff
@@ -7,5 +7,11 @@
   "short_description": "filebeat description.",
   "support": "contrib",
   "supported_os": ["linux","mac_os","windows"],
-  "version": "0.1.0"
+  "version": "0.1.0",
+  "public_title": "Datadog-Filebeat Integration",
+  "categories":["os & ystem"],
```
Comment (jeremy-lq) [inline filebeat/manifest.json]: system

File: logstash/README.md [hunk 5]
```diff
@@ -31,9 +32,12 @@ When you run `datadog-agent info` you should see something like the following:
...
 The logstash check is compatible with Logstash 5.6 and possible earlier versions. Currently it does not support the new pipelines metrics in Logstash 6.0 yet.
 
-## Metrics
+## Data Collected
+### Metrics
+See [metadata.csv](https://github.com/DataDog/integrations-extras/blob/master/filebeat/metadata.csv) for a list of metrics provided by this integration.
 
-See [metadata.csv](metadata.csv) for a list of metrics provided by this integration.
+### Events
+The Filebeat check does not include any event at this time.
```
Comment (jeremy-lq) [inline logstash/README.md]: The Logstash integration doc mentions/links to the filebeat one multiple times.

File: filebeat/README.md (hunk 1 above)
Comment (l0k0ms) [reply filebeat/README.md]: 👍

File: filebeat/README.md (hunk 3 above)
Comment (l0k0ms) [reply filebeat/README.md]: 👌

File: filebeat/README.md (hunk 2 above)
Comment (l0k0ms) [reply filebeat/README.md]: 👌

File: filebeat/manifest.json (hunk 4 above)
Comment (l0k0ms) [reply filebeat/manifest.json]: 👌

File: logstash/README.md (hunk 5 above)
Comment (l0k0ms) [reply logstash/README.md]: 👌

**Pull request merged**